main.go

```diff
@@ -10,19 +10,32 @@ import (
 	"github.com/opencontrol/fedramp-templater/templater"
 )
 
+type subCommand uint8
+
+const (
+	_ subCommand = iota // Default value. This value is used as a placeholder when creating instances of subCommands
+	diff
+	fill
+)
+
+func(cmd subCommand) isType(otherCmd subCommand) bool {
+	return cmd == otherCmd
+}
+
 type options struct {
 	openControlsDir string
 	inputPath string
 	outputPath string
-	diff bool
-	fill bool
+	cmd subCommand
 }
 
 func printUsage() {
-	log.Fatal("Usage:\n\n" +
-		"\tfedramp-templater fill <openControlsDir> <inputDoc> <outputDoc>\n\n" +
-		"\tor\n\n" +
-		"\tfedramp-templater diff <openControlsDir> <inputDoc>")
+	log.Fatal(`Usage:
+	fedramp-templater fill <openControlsDir> <inputDoc> <outputDoc>
+
+	or
+
+	fedramp-templater diff <openControlsDir> <inputDoc>`)
```

Comment:
If we want to keep it the same yeah.
This is how it looks now: 
![image](https://cloud.githubusercontent.com/assets/7788930/18012906/84877420-6b89-11e6-83cc-2ba117612a7d.png)

Compared to before (without the diff command):
![image](https://cloud.githubusercontent.com/assets/7788930/18012928/a595b2b2-6b89-11e6-89d1-da8be7db78e0.png)